pkg/chart/jupyterhub.go

```diff
@@ -91,9 +94,12 @@ func jupyterReleaseName(namespace string) string {
 	return fmt.Sprintf("%v-%v", string(gensql.ChartTypeJupyterhub), namespace)
 }
 
-func (c Client) jupyterMergeValues(team gensql.TeamGetRow, configurableValues JupyterConfigurableValues) jupyterValues {
-	// TODO: Her må vi sjekke om configurableValues er tom, og hente fra databasen hvis det er tilfelle
-	// configurableValues er tom hvis den kommer fra team-endringer eller som en sync fra admin.
+func (c Client) jupyterMergeValues(ctx context.Context, team gensql.TeamGetRow, configurableValues JupyterConfigurableValues) (jupyterValues, error) {
+	if len(configurableValues.UserIdents) == 0 {
+		if err := c.getJupyterConfigurableValues(ctx, team, &configurableValues); err != nil {
```

Comment:
Jeg liker å unngå funksjoner med sidefunksjoner, altså at vi ikke sender inn objektet som skal modifiseres. Lager en commit på dette. 
Vi må også mappe om alle de andre verdiene, for er `configurableValues` tom (for eks. når den kommer fra en admin), så har den kun teamID satt.

Reply:
Hadde glemt at `TeamConfigurableValuesGet` gjør det jeg trodde vi manglet...